contents/tutorials/flutter-feature-flags.md

```diff
@@ -0,0 +1,288 @@
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: isFlagEnabled ? Colors.green : Colors.red,
+    );
+  }
+}
+```
+
+Press **F5** and run the app in any emulator (we chose Android) to see your app in action.
+
+![Basic setup of the Flutter app](../images/tutorials/flutter-ab-tests/basic-app.png)
+
+## 2. Add PostHog to your app
+
+With our app set up, it’s time to install and set up PostHog. If you don't have a PostHog instance, you can [sign up for free](https://us.posthog.com/signup).
+
+To start, install [PostHog’s Flutter SDK](/libraries/flutter) by adding `posthog_flutter` to your `pubspec.yaml`:
+
+```yaml file=pubspec.yaml
+# rest of your code
+
+dependencies:
+  flutter:
+    sdk: flutter
+  posthog_flutter: ^4.0.1
+
+# rest of your code
+```
+
+Next, we configure PostHog in each platform using our project API key and instance address. You can find these in [your project settings](https://us.posthog.com/settings/project).
+
+### Android setup
+
+For Android, add your PostHog configuration to your `AndroidManifest.xml` file located in the `android/app/src/main`:
+
+```xml file=android/app/src/main/AndroidManifest.xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="your.package.name">
+    <application>
+        <!-- ... other configuration ... -->
+        <meta-data android:name="com.posthog.posthog.API_KEY" android:value="<ph_project_api_key>" />
+        <meta-data android:name="com.posthog.posthog.POSTHOG_HOST" android:value="<ph_instance_address>" /> <!-- usually 'https://app.posthog.com' or 'https://eu.posthog.com' -->
+        <meta-data android:name="com.posthog.posthog.TRACK_APPLICATION_LIFECYCLE_EVENTS" android:value="true" />
+        <meta-data android:name="com.posthog.posthog.DEBUG" android:value="true" />
+    </application>
+</manifest>
+```
+
+You'll also need to update the minimum Android SDK version to `21` in `android/app/build.gradle`:
+
+```gradle_kotlin file=android/app/build.gradle
+// rest of your config
+
+    defaultConfig {
+      minSdkVersion 21
+      // rest of your config
+    }
+
+// rest of your config
+```
+
+### iOS setup
+
+For iOS, you need to have [Cocoapods](https://guides.cocoapods.org/using/getting-started.html) installed. Then add your PostHog configuration to the `Info.plist` file located in the `ios/Runner` directory:
+
+```xml ios/Runner/Info.plist
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+<!-- rest of your configuration -->
+  <key>com.posthog.posthog.API_KEY</key>
+  <string><ph_project_api_key></string>
+  <key>com.posthog.posthog.POSTHOG_HOST</key>
+  <string><ph_instance_address></string>  <!--  https://app.posthog.com or https://eu.posthog.com -->
+  <key>com.posthog.posthog.CAPTURE_APPLICATION_LIFECYCLE_EVENTS</key>
+  <true/>
```

Comment:
Any reason the Android one has `com.posthog.posthog.DEBUG` and this one not? you can add here as well